lib/editor/util/index.js

```diff
@@ -60,6 +60,9 @@ export function sortAndFilterTrips (
           // only has one frequency entry (for now).
           return a.frequencies[0].startTime - b.frequencies[0].startTime
         }
+        if (a.stopTimes.length === 0 && b.stopTimes.length === 0) return 0
+        if (a.stopTimes.length === 0) return -1
+        if (b.stopTimes.length === 0) return 1
```

Comment:
This code block definitely deserves a descriptive comment describing: why it's sorting this way, why there might be a trip with no stop times, etc. It's the only change to non-test code in the whole PR, I think it's important that something like this have some comments.